metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-misp",
-  "version": "0.2.2",
+  "version": "0.2.3-rc0",
```

Comment:
it seems like you rebased against an older version of master. Can you rebase again, against the latest version?